src/guidellm/utils/constants.py

```diff
@@ -17,3 +24,9 @@
 PREFERRED_DATA_SPLITS = ["test", "validation", "train"]
 
 STANDARD_SLEEP_INTERVAL = 0.1
+
+STANDARD_REQUEST_TIMEOUT = 30
```

Comment:
What about having this in settings as `settings.requests.timeout` along with `settings.requests.sleep_interval`?

Reply:
That's fair, it would be nice to easily control that with env variables, will update these